src/google/protobuf/compiler/cpp/cpp_bootstrap_unittest.cc

```diff
@@ -183,6 +183,69 @@ TEST(BootstrapTest, GeneratedFilesMatch) {
   }
 }
 
+//test Generate in cpp_generator.cc
+TEST(BootstrapTest, optionNotExist) {
```

Comment:
@wangkirin Thanks for the pull request! I would say let's just keep this first test only. The second one seems to cover roughly the same case and I wouldn't worry about the 3rd one since it covers an internal-only option that we might remove in the future.

Reply:
OK，I will keep the first case only

Reply:
> @wangkirin Thanks for the pull request! I would say let's just keep this first test only. The second one seems to cover roughly the same case and I wouldn't worry about the 3rd one since it covers an internal-only option that we might remove in the future.

Done